src/bucket/LiveBucketList.cpp

```diff
@@ -31,7 +34,7 @@ LiveBucketList::sumBucketEntryCounters() const
     {
         for (auto const& b : {lev.getCurr(), lev.getSnap()})
         {
-            if (b->isIndexed())
+            if (!b->isEmpty())
```

Comment:
I don't quite understand this change. Where are non-indexed, non-empty buckets coming from? Are we interested in these?

Reply:
It doesn't really matter, as we have the invariant that every BucketList referenced bucket is either indexed or empty. I made this change because outside of bucket creation and GC, no class should know/care whether or not a bucket is indexed. Ideally, `isIndexed` would be private, but we have to call it in BucketManager for gc and the output iterator for initial creation. Generally speaking I think it's better for most classes to reason about a bucket being empty or non-empty instead of reasoning about whether or not it is indexed, but it's a minor point.